### PR TITLE
resolve #40 Mail service

### LINKED ISSUE #40: Mail Service
Ability to send email with `:title, :body` to selected scope of users. `TODO: specify which criterias needed for select the scope`

### PR DESCRIPTION
**Pull request by denys-medynskyi**
admin will be able to send email to all members with given subject and body. 
Email body/subject will be save and can be reused.
At this moment only text is supported.

### CODE REVIEW

File: app/controllers/admin/emails_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+module Admin
+  class EmailsController < BaseController
+    helper_method :email, :emails
+
+    def index
+    end
```
Comment (sveredyuk) [inline app/controllers/admin/emails_controller.rb]: Templates are present. Index, show and new -> redundant empty methods..

File: app/services/email/send.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class Email
+  class Send < ApplicationService
+    def initialize(email:, recipient_ids:)
+      @email = email
+      @recipient_ids = recipient_ids
+    end
+
+    def call
+      recipient_ids.each do |recipient_id|
+        EmailMailer.custom(email.id, recipient_id).deliver
```
Comment (sveredyuk) [inline app/services/email/send.rb]: deliver_later.. or you need real time sending (no)

**Review by sveredyuk — APPROVED** [review]
Please, fix my comments and merge

**Comment by denys-medynskyi** [comment]
@sveredyuk thanks for comments

**Pull request merged**